packages/helpers/src/replacement-variables/extractExcerptFromContent.js

```diff
@@ -0,0 +1,29 @@
+import { stripFullTags as stripTags } from "../strings/stripHTMLTags";
```

Comment:
1. This helper is copied from `@yoast/wordpress-seo/packages/js/src/helpers/replacementVariableHelpers.js`. It was decided that it's not a good idea to use the helper from `@yoast/wordpress-seo` inside `seo-integration` package, since we want the package to be platform agnostic. Hence, the decision to create a separate helper inside `@yoast/helpers` package, as this package already exists as a dependency in `seo-integration`.

2. `stripTags` helper is imported from `stripHTMLTags.js` instead of `@wordpress/sanitize` package as it turns out that the package is not available as a stand-alone package.